frontend/src/layouts/SqlEditorLayout.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <div class="relative h-screen overflow-hidden flex flex-col">
+    <template v-if="isDemo">
+      <BannerDemo />
+    </template>
+    <template v-else-if="isReadonly">
```

Comment:
I think we only allow readonly/SELECT query?

Reply:
add the notification if server is in readonly mode